app/pb_kits/playbook/pb_tooltip/_tooltip.scss

```diff
@@ -0,0 +1,103 @@
+@import "../tokens/positioning";
+@import "../tokens/colors";
+
+@keyframes fadeIn {
+  from {
+    opacity: 0;
+  }
+
+  to {
+    opacity: 1;
+  }
+}
+@keyframes fadeOut {
+  from {
+    opacity: 1;
+  }
+
+  to {
+    opacity: 0;
+  }
+}
+
+[class^="pb_tooltip_kit"] {
+  .tooltip_tooltip {
+    display: none;
+    opacity: 0;
+    left: 0;
+    animation-name: fadeIn;
+    animation-duration: 150ms;
+    animation-timing-function: linear;
+    animation-fill-mode: forwards;
+
+    &.flipped {
+      margin-top: 15px;
+      .arrow {
+        top: -35%;
+        border-color: transparent transparent white transparent;
+      }
+    }
+
+    &.react {
+      .arrow {
+        top: 78%;
+      }
+      &.flipped {
+        .arrow {
+          top: -8%;
+        }
+      }
+    }
+
+    .arrow {
+      content: " ";
+      position: absolute;
+      top: 100%;
+      left: 50%;
+      border-color: white transparent transparent transparent;
+      border-style: solid;
+      border-width: 10px;
+      margin-bottom: 50px;
+      margin-left: -10px;
+    }
+    &.show {
+      opacity: 1;
+      filter:alpha(opacity=1);
+      display: block;
+      z-index: $z_9;
+      margin-bottom: $space_sm;
+      background-color: white;
+      padding: $space_xs $space_sm $space_xs $space_sm;
+      box-shadow: $shadow_deeper;
+      border-radius: 3px;
+      &.fade_out {
+        animation-name: fadeOut;
+        animation-duration: 150ms;
+        animation-timing-function: linear;
+        animation-fill-mode: forwards;
+      }
+    }
+  }
+  &[class*=_dark]{
+    .tooltip_tooltip{
+      color: $white;
+
+      &.show {
+        opacity: 85%;
+        background-color: black;
```

Comment:
```suggestion
        background-color: rgba(#000, $opacity_9);
```

Reply:
Even better would be if you can add `$black: #000000` to our color variables as well